src/Symfony/Bundle/FrameworkBundle/Templating/TemplateNameParser.php

```diff
@@ -74,6 +74,12 @@ public function parse($name)
 
     private function isAbsolutePath($file)
     {
-        return (bool) preg_match('#^(?:/|[a-zA-Z]:)#', $file);
+        $isAbsolute = (bool) preg_match('#^(?:/|[a-zA-Z]:)#', $file);
+
+        if ($isAbsolute) {
+            @trigger_error('Absolute template path support is deprecated since Symfony 3.1 and will throw an exception in 4.0.', E_USER_DEPRECATED);
```

Comment:
Instead of:

```
... and will throw an exception in 4.0
```

I think we usually write it like this:

```
...and it will be removed in 4.0
```